src/env_node.ml

```diff
@@ -126,7 +126,7 @@ let rec inline_tests t ~profile =
       | {inline_tests = None; _} ->
         begin match t.inherit_from with
         | None ->
-          if profile = "release" then
+          if Profile.is_release profile then
```

Comment:
How about a `Profile.with_inline_test: t -> bool` instead here?

If we ever want to add new profiles or to allow disabling inline tests in user defined profiles it would make things easier.

Reply:
Yeah that's a good point. I didn't want to introduce any non essential refactoring, but I can squeeze this one as a separate commit.